lib/modules/manager/crossplane/readme.md

```diff
@@ -0,0 +1,40 @@
+The `crossplane` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
+This is because there is no commonly accepted file/directory naming convention for crossplane YAML files and we don't want to check every single `*.yaml` file in repositories just in case any of them have Crossplane Packages definitions: Configurations, Providers, Functions.
+
+If most `.yaml` files in your repository are Crossplane ones, then you could add this to your config:
+
+```json
+{
+  "crossplane": {
+    "fileMatch": ["\\.yaml$"]
+  }
+}
+```
+
+If instead you have them all inside a `packages/` directory, you would add this:
+
+```json
+{
+  "crossplane": {
+    "fileMatch": ["packages/.+\\.yaml$"]
+  }
+}
+```
+
+Or if it's only a single file then something like this:
+
+```json
```

Comment:
Can you add a codeblock title, and drop the introductory sentences?

```
json title="fileMatch for a single Crossplane file"
```

[Material for MkDocs manual, adding a title to codeblocks](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title)